assets/javascripts/helpers/ifCond.js.es6

```diff
@@ -5,10 +5,10 @@ import { registerHelper } from 'discourse-common/lib/helpers';
 
 registerHelper('ifCond', function(params) {
     let topicValueIsGreaterThanTheBoundaryRiskLevel = false;
-    let topicValue = params[0];
+    let topicValue = parseInt(params[0]);
     let operator = params[1];
-    let boundaryRiskLevel = params[2];
-    if (operator === '>'){
+    let boundaryRiskLevel = parseInt(params[2]);
+    if (operator === '>='){
         if (topicValue > boundaryRiskLevel) {
```

Comment:
This should also be ">=" I think, unless my logic is fuzzy

Reply:
argh.. silly mistakes.  :( fixed it now